src/server/room-controller.js

```diff
@@ -103,14 +103,18 @@ export default class RoomController {
 
   async handleAuthorizeResourcesAccess(req, res) {
     const { roomId } = req.params;
-    const { _id: userId } = req.user;
+
+    const userId = req.user?._id || null;
```

Comment:
Small thing, due to the early return and "falsy" check, `|| null` is not needed.

Reply:
Fixed, thanks